ppml/trusted-big-data-ml/scala/docker-occlum/run_spark_on_occlum_glibc.sh

```diff
@@ -122,7 +129,27 @@ build_spark() {
     mkdir -p image/bin/jars
     cp -f $BIGDL_HOME/jars/* image/bin/jars
     cp -rf /opt/spark-source image/opt/
-    occlum build
+    occlum build --image-key /opt/occlum_spark/data/image_key
+    build_initfs
```

Comment:
Need to add ATTESTATION env. Default is false. In that case, we can skip ra_tls

Reply:
Updated here https://github.com/intel-analytics/BigDL/pull/4507/files#diff-289171a84c58724029266cd84d480d3a7e4b7718022baa17b9a5866209c2b836R20

Reply:
I think this is not secure.